### PR TITLE
Upgrade to podman 5.0.0

### PR DESCRIPTION
**Pull request by djgraff209**
Upgrade to latest release of podman v5.0.0

### CODE REVIEW

File: tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $packageName    = 'podman-cli'
-$url            = 'https://github.com/containers/podman/releases/download/v4.9.3/podman-remote-release-windows_amd64.zip'
+$url            = 'https://github.com/containers/podman/releases/download/v5.0.0/podman-remote-release-windows_amd64.zip'
 $checksum       = 'ed2d0c91f35e207dcef5eddbb00b2641d0c81ed6045711160bcce7a9fbd30d7e'
```
Comment (lareeth) [inline tools/chocolateyInstall.ps1]: The checksum needs to be updated, are you able to update it? They provide the checksums as an addition file on the GitHub release

Comment (djgraff209) [reply tools/chocolateyInstall.ps1]: Yes give me about 15 minutes

Comment (djgraff209) [reply tools/chocolateyInstall.ps1]: Added

**Comment by lareeth** [comment]
LGTM

**Comment by djgraff209** [comment]
Thanks for the quick merge - how long before this will show up as an upgrade in chocolatey?

**Comment by lareeth** [comment]
Its published automatically, it has to go through the chocolatey validation process before being available, but you can manually install it if you're happy to trust it

https://community.chocolatey.org/packages/podman-cli/5.0.0

**Pull request merged**